ugvc/mrd/srsnv_plotting_utils.py

```diff
@@ -1044,13 +1044,13 @@ def plot_subsets_hists(
         name = item
 
         plt.figure(figsize=(8, 6))
-
+        logflag = False
         for label in labels_dict:
-            _ = (
-                td[td["label"] == label][score]
-                .clip(upper=max_score)
-                .hist(bins=bins, label=labels_dict[label], alpha=0.8, density=True)
-            )
+            h, bin_edges = np.histogram(td[td["label"] == label][score].clip(upper=max_score), bins=bins, density=True)
+            bin_centers = (bin_edges[1:] + bin_edges[:-1]) / 2
+            plt.bar(bin_centers, h, label=labels_dict[label], alpha=0.8, width=1, align="center")
+            if any(h > 0):
```

Comment:
can you explain what does positive h means?

Reply:
in case there is no h>0 the log scale fails and it crashes the module